web_view_leaflet_map/i18n/fr.po

```diff
@@ -119,3 +54,30 @@ msgstr "Vue"
 #: model:ir.model.fields,field_description:web_view_leaflet_map.field_ir_ui_view__type
 msgid "View Type"
 msgstr "Type de Vue"
+
+#~ msgid "Activity"
+#~ msgstr "Activité"
+
+#~ msgid "Calendar"
+#~ msgstr "Calendrier"
```

Comment:
All that addition looks weird. Why add commented translation ? 
could you revert changes in fr.po ? 
thanks !

Reply:
Yes, changes only in map_renderer.js